modules/docs/app/VoloDocs.Web/abp.resourcemapping.js

```diff
@@ -7,6 +7,7 @@
         "@libs"
     ],
     mappings: {
-
+        "@node_modules/@microsoft/signalr/dist/browser/signalr.js": "@libs/signalr/",
+        "@node_modules/@microsoft/signalr/dist/browser/signalr.js.map": "@libs/signalr/"
```

Comment:
We can use the `@abp/signalr` package instead of the manual installation. See https://docs.abp.io/en/abp/latest/SignalR-Integration#asp-net-core-mvc-razor-pages-ui